doc/api/pyplot_summary.rst

```diff
@@ -40,79 +40,178 @@ Managing Figure and Axes
 Adding data to the plot
 -----------------------
 
+Basic
+^^^^^
```

Comment:
```suggestion
Core
^^^^
```

less value loaded that "basic"?

Reply:
I've reused the "Plotting" subesction titles from https://matplotlib.org/stable/api/axes_api.html#plotting. I'm happy choose better names, but we should do this in both places.

To me "Core" has a connotation that *we* have decided that these are more important than the others. This is not my intention. The only point about this section is that these are some simple (basic) and common plots. How about "fundamental"?

Reply:
ef583fc4591995a999e615e93513624394e15d35 <- and I am the blame on that name in that file 🤦🏻 .  I am sold on same name in both and happy to defer that.